script/release/01.push-release-branches-DSL.txt

```diff
@@ -0,0 +1,66 @@
+def cloneScript_01 = 
```

Comment:
@mbiarnes I completely agree we need to store the DSL files in git repo. We just need to avoid hard-coding the paths (to make it as much resilient and stable as possible). I'll take a closer look at this today or tomorrow to see how we can easily do that.